minimesos/src/main/java/com/containersol/minimesos/state/State.java

```diff
@@ -23,6 +26,13 @@ public static State fromJSON(String jsonString) throws JsonParseException, JsonM
         }
     }
 
+    private Map<String, String> flags = new HashMap<>();
```

Comment:
Yep, class members should be listed before constructors.... isn't it annoying? ;-)